01.md

```diff
@@ -92,7 +92,7 @@ Kinds specify how clients should interpret the meaning of each event and the oth
 
 And also a convention for kind ranges that allow for easier experimentation and flexibility of relay implementation:
 
-- for kind `n` such that `1000 <= n < 10000 || 4 <= n < 45 || n == 1 || n == 2`, events are **regular**, which means they're all expected to be stored by relays.
+- for kind `n` such that `4 <= n < 10000 || n == 1 || n == 2`, events are **regular**, which means they're all expected to be stored by relays.
```

Comment:
Fine with me, do you think it makes it more readable?

Reply:
Every time I read this it takes me a while to realize that the range includes everything other than 0 and 3. Might as well say it explicitly, especially since the opposite range is defined just below in the same way.